core/templates/core/500.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'core/base.html' %}
+{% block content %}
+{% load static %}
+<div class="container">
+    <div class="error404-message" style="font-size: 36px;">
+        UwU Oopsie Whoopsie. We did a fucky wucky, a fucko boingo.
+        <br>
+        Our engineers are working vewwwy hard to fix this. XD
```

Comment:
everything looks good besides these lines but it kinda geeks me lol

Reply:
Just fixed this up.